tools/ci-build/Dockerfile

```diff
@@ -121,17 +121,14 @@ ARG wasm_pack_version=0.11.0
 RUN cargo install wasm-pack --locked --version ${wasm_pack_version}
 
 FROM install_rust AS wasmtime
-ARG wasmtime_precompiled_url=https://github.com/bytecodealliance/wasmtime/releases/download/v7.0.0/wasmtime-v7.0.0-x86_64-linux.tar.xz
-ARG wasmtime_precompiled_sha256=b8a1c97f9107c885ea73a5c38677d0d340a7c26879d366e8a5f3dce84cffec99
-RUN set -eux; \
-    curl "${wasmtime_precompiled_url}" -L -o wasmtime.xz; \
-    echo "${wasmtime_precompiled_sha256} wasmtime.xz" | sha256sum --check; \
-    tar xf wasmtime.xz; \
-    mv wasmtime-v*/wasmtime /opt;
+ARG cargo_wasmtime_version=17.0.0
```

Comment:
If possible, it would be nice to use precompiled binaries to reduce image build time.

Reply:
Looks like wasmtime does offer [precompiled binaries](https://github.com/bytecodealliance/wasmtime/releases/tag/v17.0.1) for their tagged releases so this should be possible. I'll see if I have time to fit it in after updating the canary